diff --git a/conn.go b/conn.go
index 209eddf..7405222 100644
--- a/conn.go
+++ b/conn.go
@@ -30,6 +30,7 @@ var (
                "com.datastax.bdp.cassandra.auth.DseAuthenticator",
                "io.aiven.cassandra.auth.AivenAuthenticator",
                "com.ericsson.bss.cassandra.ecaudit.auth.AuditPasswordAuthenticator",
+               "com.amazon.helenus.auth.HelenusAuthenticator",
        }
 )

@@ -1441,7 +1441,7 @@ func (c *Conn) awaitSchemaAgreement(ctx context.Context) (err error) {
 func (c *Conn) localHostInfo(ctx context.Context) (*HostInfo, error) {
        row, err := c.query(ctx, "SELECT * FROM system.local WHERE key='local'").rowMap()
        if err != nil {
-               return nil, err
+               //return nil, err
        }

        port := c.conn.RemoteAddr().(*net.TCPAddr).Port
